### PR TITLE
[ARCHETYPE-654] Clarify used repository/server ids for catalog "remote"

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: maven-archetype-plugin/src/site/fml/faq.fml [hunk 1]
```diff
@@ -35,11 +35,13 @@ under the License.
     </faq>
 
     <faq id="authentication">
-      <question>How to generate a project from an archetype in an authenticated repository?</question>
+      <question>How to generate a project from an archetype in a custom (potentially authenticated) repository?</question>
       <answer>
-        <p>The server id used to download the artifact is <code><i>[archetypeArtifactId]</i>-repo</code>: you have to
```
Comment (kwin) [inline maven-archetype-plugin/src/site/fml/faq.fml:40]: this repo id is format is still used in https://github.com/apache/maven-archetype/blob/3a2b725198c3823fd5d7e9f88b665b2e4515a202/archetype-common/src/main/java/org/apache/maven/archetype/generator/DefaultArchetypeGenerator.java#L70 (i.e. for downloading the actual archetype (not the catalog)), not sure if it just acts as a fallback though.

Comment (kwin) [reply maven-archetype-plugin/src/site/fml/faq.fml:40]: clarified now in https://github.com/apache/maven-archetype/pull/185/commits/31c6734679734825b9041e3cf6e1519fd06343de.

**Comment by jira-importer** [comment]
Resolve #654

**Comment by jira-importer** [comment]
Resolve #654

**Pull request merged**